File: src/main/java/com/summer/lijiahao/utils/openapi/actions/SettingAction.java
package com.summer.lijiahao.utils.openapi.actions;

import com.intellij.icons.AllIcons;
import com.intellij.openapi.actionSystem.AnActionEvent;
import com.intellij.openapi.project.DumbAwareAction;
import com.summer.lijiahao.utils.openapi.ui.SettingDialog;
import org.jetbrains.annotations.NotNull;

public class SettingAction
        extends DumbAwareAction {
    public SettingAction() {
        super("OpenAPI配置", "OpenAPI配置内容", AllIcons.Actions.InlayGear);

    }


    public void actionPerformed(@NotNull AnActionEvent e) {
        (new SettingDialog(e.getProject())).setVisible(true);
    }
}

File: src/main/kotlin/com/summer/lijiahao/nccdevtool/opanapi/action/SettingAction.kt
package com.summer.lijiahao.nccdevtool.opanapi.action

import com.intellij.icons.AllIcons
import com.intellij.openapi.actionSystem.AnActionEvent
import com.intellij.openapi.project.DumbAwareAction
import com.summer.lijiahao.nccdevtool.opanapi.ui.OpenapiSettingDialog

class SettingAction : DumbAwareAction("OpenAPI配置", "OpenAPI配置内容", AllIcons.Actions.InlayGear) {
    override fun actionPerformed(e: AnActionEvent) {
        val dialog = OpenapiSettingDialog(e)
        dialog.setSize(500, 300)
        dialog.show()
    }
}